### PR TITLE
Add handsontable

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@mwcraig : It fails for osx. Should I skip the osx build in meta.yaml or commit with [skip travis-ci] message?

**Comment by mwcraig** [comment]
I'm not super-familiar with javascript packages but it looks like it fails for a reason we shouldn't ignore...it needs rimraf but isn't finding it. Not sure why it passes on linux and windows, tbh.

@conda-forge/core -- how should we handle javascript packaging? Do we want to go down the road of rebuilding everything on npm? 

If we want to start building out everything on npm that will mean a lot of translating dependencies from `package.json` files.

For example, the list of dependencies for this package, `handsontable`, is at https://github.com/handsontable/handsontable/blob/master/package.json#L57

For us to build the package we need the `devDependencies` as build dependencies and the `dependencies` as run dependencies if we want the conda packages to properly reflect the dependencies...so really this recipe should have much more extensive dependencies, as should other recently merged javascript packages (I'm guessing...and FWIW I did some of that merging).

Thoughts?

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge. Although travis check didn't pass, it should be good to merge because it is a noarch.

File: recipes/handsontable/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+source:
+  url: https://github.com/handsontable/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
```
Comment (mwcraig) [inline recipes/handsontable/meta.yaml]: This is missing dependencies; see: https://github.com/handsontable/handsontable/blob/master/package.json#L57

**Comment by dougalsutherland** [comment]
The `rimraf` thing I _think_ is just a problem in the build script, shouldn't be an issue in the installed package: `rimraf` is just a javascript version of `rm`, basically. (Though I don't know why this was an error on Travis and not CircleCI.)

File: recipes/handsontable/meta.yaml (hunk 1 above)
Comment (dougalsutherland) [reply recipes/handsontable/meta.yaml]: For posterity: not actually an issue, see https://github.com/conda-forge/staged-recipes/wiki/Recipes-that-bundle-javascript-libraries.

**Comment by mwcraig** [comment]
Thanks for taking care of this one, @dougalsutherland

**Pull request merged**